main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
+//  Noun: Tape Machine
 //  action 1:
-
+tapeMachine.record();
 //  action 2:
-
+tapeMachine.play();
 //  action 3:
- 
+tapeMachine.rewind();
 //  2)
-//  Noun:
+//  Noun: Player1
 //  action 1:
-
+player1.run();
 //  action 2:
-
+player1.dodge();
 //  action 3:
- 
+player1.jump();
 //  3)
-//  Noun:
+//  Noun: Dog
 //  action 1:
-
+dog.bark();
 //  action 2:
-
+dog.growl();
 //  action 3:
- 
+dog.lick();
 //  4)
-//  Noun:
+//  Noun: Guitar
 //  action 1:
-
+guitar.strum();
 //  action 2:
-
+guitar.tune();
 //  action 3:
- 
+guitar.shred();
```

Comment:
`guitarPlayer`'s do all of these.  `guitar`'s do none of these things.
- [ ] revise